### PR TITLE
docs: added supported named parameter formats

### PR DESCRIPTION
**Pull request by bighappyworld**

### CODE REVIEW

File: docs/queries.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ db.query(
 });
 ```
 
-`query` takes named parameters as well:
+`query` takes named parameters as well accepting `$*name*` with `*` being any of the following pairs `{}` `[]` `()` `\\` `<>` :
```
Comment (vitaly-t) [inline docs/queries.md]: It doesn't support $\name\, but supports $/name/

Comment (bighappyworld) [reply docs/queries.md]: thanks, adjusted to forward slash

Comment (vitaly-t) [reply docs/queries.md]: Punctuation is off also. A comma is needed after ‘as wel’ and before ‘with’.

**Comment by dmfay** [comment]
@bighappyworld I can't pull a commit from a deleted repository to credit you, but you're right that that could be clearer. I've updated the docs, thanks for bringing it up!

**Pull request closed without merging**